File: libcommon/src/main/java/com/soaic/libcommon/glide/GlideSimpleLoader.java
package com.soaic.libcommon.glide;

import android.content.Context;
import android.net.Uri;

import com.soaic.libcommon.weiget.imagewatcher.ImageWatcher;

public class GlideSimpleLoader implements ImageWatcher.Loader {
    @Override
    public void load(Context context, Uri uri, ImageWatcher.LoadCallback lc) {
        GlideUtil.display(context, uri, lc);
    }
}

File: app/src/main/java/com/soaic/toolsapp/util/GlideSimpleLoader.kt
package com.soaic.toolsapp.util

import android.content.Context
import android.net.Uri
import com.soaic.libcommon.glide.GlideUtil
import com.soaic.libcommon.weiget.imagewatcher.ImageWatcher

class GlideSimpleLoader: ImageWatcher.Loader {

    override fun load(context: Context?, uri: Uri?, lc: ImageWatcher.LoadCallback?) {
        GlideUtil.display(context, uri, lc)
    }

}